samples/csharp/common/AutoML/ConsoleHelper.cs

```diff
@@ -109,6 +123,11 @@ internal static void PrintRegressionMetricsHeader()
             CreateRow($"{"",-4} {"Trainer",-35} {"RSquared",8} {"Absolute-loss",13} {"Squared-loss",12} {"RMS-loss",8} {"Duration",9}", Width);
         }
 
+        internal static void PrintRankingMetricsHeader()
+        {
+            CreateRow($"{"",-4} {"Trainer",-14}, {"nDCG@1",9}, {"nDCG@2",9}, {"nDCG@10",9}, {"DCG@1",9}, {"Duration",9}", Width);
```

Comment:
Printing NDCG@1,3,10 & DCG@10
```suggestion
            CreateRow($"{"",-4} {"Trainer",-14}, {"nDCG@1",9}, {"nDCG@3",9}, {"nDCG@10",9}, {"DCG@10",9}, {"Duration",9}", Width);
```